string mex_id
int8 mex_new_status
---
bool success